tools/hooks/restrict_levelbuilder_changes.rb

```diff
@@ -3,7 +3,7 @@
 REPO_DIR = File.expand_path('../../../', __FILE__)
 LEVELS_DIR = File.expand_path(REPO_DIR + '/dashboard/config/scripts', __FILE__)
 COURSES_DIR = File.expand_path(REPO_DIR + '/dashboard/config/courses', __FILE__)
-WHITELISTED_FILES = %w[dashboard/config/locales/dsls.en.yml dashboard/config/locales/scripts.en.yml dashboard/config/locales/courses.en.yml].map {|f| File.join(REPO_DIR, f)}
+WHITELISTED_FILES = %w[dashboard/config/locales/dsls.en.yml dashboard/config/locales/scripts.en.yml dashboard/config/locales/courses.en.yml dashboard/config/locales/unplugged.en.yml].map {|f| File.join(REPO_DIR, f)}
```

Comment:
nit: now that this list is getting long enough to go well beyond 100 cols, please break into multiple lines